### PR TITLE
Prepare for release 24.3.1

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 24.3.1
+
+New error codes:
+* Y064: Use simpler syntax to define final literal types.
+  For example, use `x: Final = 42` instead of `x: Final[Literal[42]]`
```
Comment (AlexWaygood) [inline CHANGELOG.md:7]: #469 didn't have a changelog entry

**Comment by AlexWaygood** [comment]
I'll hold off until #475 has landed :)

File: CHANGELOG.md (hunk 1 above)
Comment (srittau) [reply CHANGELOG.md:7]: I guess this was already part of 24.3.0? In that case, I'd just add this entry there retroactively, and maybe add a comment that the changelog entry was missing to the latest version.

Comment (AlexWaygood) [reply CHANGELOG.md:7]: No, it hasn't been included in any release yet

**Review by srittau — APPROVED** [review]

**Comment by AlexWaygood** [comment]
https://pypi.org/project/flake8-pyi/24.3.1/ 🎉

**Pull request merged**